src/bloaty.cc

```diff
@@ -1386,57 +1389,55 @@ class Bloaty {
   std::vector<ConfiguredDataSource*> sources_;
   std::vector<std::string> source_names_;
 
-  std::vector<std::unique_ptr<ObjectFile>> input_files_;
-  std::vector<std::unique_ptr<ObjectFile>> base_files_;
-  std::map<std::string, std::unique_ptr<ObjectFile>> debug_files_;
+  std::vector<std::pair<std::string, std::string>> input_files_;
```

Comment:
Please create a struct so accesses can use descriptive names instead of `first`, `second`.

Reply:
Will do.